src/booking/containers/booking-date/booking-date.component.scss

```diff
@@ -0,0 +1,6 @@
+/deep/.date-head h1 {
```

Comment:
As discussed, I don't think we should be overriding GDS guidelines and rolling our own css, however this is the first time we are using the common lib date component and without a refactor of the component itself we will not be able to add the correct styling for the date component. In future we should move toward using the gov-ui components in the common lib.

Reply:
@tarikdogan-hmcts - I'm not entirely clear which `<h1>` element is being targeted. The `<xuilib-gov-uk-date>` component itself does not contain any `<h1>` tags, but it does use `<xuilib-gov-uk-fieldset>` and `<xuilib-gov-label>`, both of which contain `<h1>` tags themselves. Which of these is your target?

Reply:
Marking as resolved, given that this CSS rule is no longer required.